src/components/accounts/AccountSettings.js

```diff
@@ -16,7 +16,7 @@ class AccountSettings extends Component {
             );
         }
         return (
-            <div className='container page'>
+            <div className='container-page'>
```

Comment:
This isn't a typo, it's actually supposed to say `container page`. Container is a Bootstrap CSS selector, and `page` is a custom selector we wrote if I recall correctly.